integration/default_app_test.go

```diff
@@ -0,0 +1,84 @@
+package integration_test
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+func testDefaultApp(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect     = NewWithT(t).Expect
+		Eventually = NewWithT(t).Eventually
+
+		pack   occam.Pack
+		docker occam.Docker
+	)
+
+	it.Before(func() {
+		pack = occam.NewPack().WithVerbose().WithNoColor()
+		docker = occam.NewDocker()
+	})
+
+	context("default_app", func() {
+		var (
+			image     occam.Image
+			container occam.Container
+
+			name   string
+			source string
+		)
+
+		it.Before(func() {
+			var err error
+			name, err = occam.RandomName()
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		it.After(func() {
+			Expect(docker.Container.Remove.Execute(container.ID)).To(Succeed())
+			Expect(docker.Image.Remove.Execute(image.ID)).To(Succeed())
+			Expect(docker.Volume.Remove.Execute(occam.CacheVolumeNames(name))).To(Succeed())
+			Expect(os.RemoveAll(source)).To(Succeed())
+		})
+
+		it("builds and runs", func() {
+			var err error
+			var logs fmt.Stringer
+
+			source, err = occam.Source(filepath.Join("testdata", "default_app"))
+			Expect(err).NotTo(HaveOccurred())
+
+			image, logs, err = pack.Build.
+				WithPullPolicy("never").
+				WithBuildpacks(
+					buildpacks.PhpDist,
+					buildpacks.Composer,
+					buildpacks.ComposerInstall,
+					buildpacks.PhpWeb,
+				).
+				Execute(name, source)
+			Expect(err).ToNot(HaveOccurred(), logs.String)
+
+			Expect(logs).To(ContainLines(ContainSubstring("Paketo PHP Distribution Buildpack")))
+			Expect(logs).To(ContainLines(ContainSubstring("Paketo Composer Buildpack")))
+			Expect(logs).To(ContainLines(ContainSubstring("Paketo Composer Install Buildpack")))
+			Expect(logs).To(ContainLines(ContainSubstring("Paketo PHP Web Buildpack")))
```

Comment:
May want to add a TODO note somewhere to update this test to not use PHP. Web since it'll be getting ripped out in favor of the newly written buildpacks. A few of them have already been released for use depending on your use case.